src/meta/service/src/configs/outer_v0.rs

```diff
@@ -483,6 +486,10 @@ pub struct RaftConfig {
     /// if not, the default name is used
     #[clap(long, default_value = "foo_cluster")]
     pub cluster_name: String,
+
+    /// Max timeout(in milli seconds) when waiting a cluster leader.
+    #[clap(long, default_value = "10000")]
+    pub wait_leader_timeout: u64,
```

Comment:
Previously this timeout is 70 seconds(election timeout is heartbeat_interval*7, while this timeout is election_timeout*10). Let's keep the default value unchanged.